src/starknet.rs

```diff
@@ -101,15 +101,19 @@ pub struct TxInfo {
 }
 
 #[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
+#[repr(C, align(16))]
```

Comment:
Is this `repr(C)` necessary?

Reply:
I think yes, the layout (field order) otherwise is undefined, and since we use this on the FFI boundary we should use this. (you could make it fail right now probably with `-Zrandomize-layout`)